src/shared/creds-util.c

```diff
@@ -703,7 +703,9 @@ int encrypt_credential_and_warn(
                               &tpm2_blob, &tpm2_blob_size,
                               &tpm2_policy_hash, &tpm2_policy_hash_size,
                               &tpm2_pcr_bank,
-                              &tpm2_primary_alg);
+                              &tpm2_primary_alg,
+                              /* ret_srk_buf= */ NULL,
```

Comment:
Is there a specific reason why for systemd-creds we're not storing the sealing key under the SRK?  It would benefit from the same benefits AFAIK

Reply:
I can't recall any specific reason, but I didn't fully study those code paths and just kept them at their current state.